### DISCUSSION TITLE
Render to opengl

### DISCUSSION THREAD

**Discussion by termiboros** [zed-industries/zed#33805] [Zed GPUI]
### Describe the feature

My machine could not able to run the zed because of vulkan drivers error, i would suggest that adding opengl would work fine on my machine and other

**Comment by someone13574** [comment]
There is an open PR (#13114) which fixes OpenGL rendering. Once it is merged you can compile with `RUSTFLAGS="--cfg gles"`. Adding a fallback mode would need to be done in blade.

**Comment by lukaslihotzki** [comment]
Blade's part of the fallback mode is tracked in https://github.com/kvark/blade/issues/130. The macOS renderer does not use blade, so support for non-Metal macs is a separate issue.

**Comment by andreymal** [comment]
Someone needs to fix crashes in blade https://github.com/kvark/blade/issues/133#issuecomment-2580992184
